sdk/spring/spring-cloud-azure-feature-management/README.md

```diff
@@ -192,10 +192,159 @@ feature-management:
+
+**Note:** `start` and `end` must be both specified to enable `recurrence`.
+
+```yaml
+feature-management:
+  feature-flags:
+    feature-v:
+      enabled-for:
+        -
+         name: TimeWindowFilter
+          parameters:
+            start: "Fri, 22 Mar 2024 20:00:00 GMT",
+            end: "Sat, 23 Mar 2024 02:00:00 GMT",
+            recurrence:
+              pattern:
+                type: "Daily",
+                interval: 1
+              range:
+                  type: "Numbered"
+                  numberOfOccurrences: 3
+```
+
+The `recurrence` settings is made up of two parts: `pattern` (how often the time window will repeat) and `range` (for how long the recurrence pattern will repeat). To create a recurrence rule, you must specify both `pattern` and `range`. Any pattern type can work with any range type.
+
+**Advanced:** The time zone offset of the `start` property will apply to the recurrence settings.
+
+#### Recurrence Pattern
+
+There are two possible recurrence pattern types: `Daily` and `Weekly`. For example, a time window could repeat "every day", "every 3 days", "every Monday" or "every other Friday".
+
+* Parameters
+
+  Property | Relevance | Description
+  -----------|-------|-----
+  **type** | Required  | Two valid values: `Daily`, `Weekly`.
+  **interval** | Optional  | Specifies the interval between each start time of occurrence. Default value is 1. <br/>For example, if the interval is 2 with `Daily` type, and current occurrence is 2:00 AM ~ 3:00 AM on 2024/05/11, then the next occurrence should be 2:00 AM ~ 3:00 AM on 2024/05/13
+  **daysOfWeek** | Optional  | Specifies on which day(s) of the week the event occurs. It's required when `Weekly` type, not applicable for `Daily` type. 
+  **firstDayOfWeek** | Optional  | Specifies which day is considered the first day of the week. Default value is `Sunday`. Not applicable for `Daily` type. 
+  
+  * Example
+    * Daily
+  
+        The daily recurrence pattern causes the time window to repeat based on interval between each occurrence.
+
+        The following example will repeat from 2:00 AM to 3:00 AM on every 2 days
+
+        ```yaml
+        start: "Mon, 13 May 2024 02:00:00 GMT",
+        end: "Mon, 13 May 2024 03:00:00 GMT",
+        recurrence:
+          pattern:
+            type: "Daily",
+            interval: 2
+          range:
+            type: "NoEnd"
+        ```
+
+    * Weekly
+  
+      The weekly recurrence pattern causes the time window to repeat on the same day or days of the week, based on the number of weeks between each set of occurrences.
+
+      The following example will repeat from 2:00 AM to 3:00 AM on every other Monday and Tuesday
+
+        ```yaml
+        start: "Mon, 13 May 2024 02:00:00 GMT",
+        end: "Mon, 13 May 2024 03:00:00 GMT",
+        recurrence:
+          pattern:
+            type: "Weekly",
+            interval: 2,
+            daysOfWeek: 
+              - Monday
+              - Tuesday
```

Comment:
Have you tried using this yaml. I get a json parsing error when trying to use it.

Reply:
Sorry, updated